### PR TITLE
ReviewConsensusVariants updates

### PR DESCRIPTION
**Pull request by nh13**
Adds NA for empty filters: this makes it easier for Unix utilities to consume.

Properly format the output base

### CODE REVIEW

File: src/main/scala/com/fulcrumgenomics/umi/ReviewConsensusVariants.scala [hunk 1]
```diff
@@ -253,21 +253,27 @@ class ReviewConsensusVariants
...
+            filters  = filters,
             A = consensusCounts.a,
             C = consensusCounts.c,
             G = consensusCounts.g,
             T = consensusCounts.t,
             N = consensusCounts.n,
             consensus_read = consensusReadName,
             consensus_insert = toInsertString(rec),
-            consensus_call = c.getReadBase.toString.toUpperCase,
+            consensus_call = c.getReadBase.toChar.toUpper,
```
Comment (nh13) [inline src/main/scala/com/fulcrumgenomics/umi/ReviewConsensusVariants.scala]:
```
scala> 84.toByte.toString.toUpperCase
res0: String = 84
scala> 84.toChar.toUpper
res1: Char = T
```

File: src/main/scala/com/fulcrumgenomics/umi/ReviewConsensusVariants.scala [hunk 2]
```diff
@@ -253,21 +253,27 @@ class ReviewConsensusVariants
           val mi = toMi(rec)
           val consensusReadName = c.getRecord.getReadName + readNumberSuffix(rec)
           val rawCounts = BaseCounts(rawByMiAndReadNum(mi + readNumberSuffix(rec)))
-
-          val m =  ConsensusVariantReviewInfo(
+          val filters = {
+            variant.filters.getOrElse("") match {
+              case "" => "NA"
```
Comment (tfenne) [inline src/main/scala/com/fulcrumgenomics/umi/ReviewConsensusVariants.scala]: I don't love this.  I usually pop these files open in excel, and it's much easier to differentiate empty vs. non-empty.

Also, if we're going to put anything here I would think `PASS` or `.` to mirror what VCF does would be clearer.

Comment (tfenne) [reply src/main/scala/com/fulcrumgenomics/umi/ReviewConsensusVariants.scala]: Also, isn't this a much more verbose way of writing `variant.filters.getOrElse("NA")`?  If so, you could just do that and move it back inline.  Of it `variant.filters` sometimes contains `Some("")` then I'd say we should fix that and or do `variant.filters.filter(_.nonEmpty).getOrElse("NA")`.

File: src/main/scala/com/fulcrumgenomics/umi/ReviewConsensusVariants.scala (hunk 1 above)
Comment (tfenne) [reply src/main/scala/com/fulcrumgenomics/umi/ReviewConsensusVariants.scala]: Huh, how did that ever work?

File: src/main/scala/com/fulcrumgenomics/umi/ReviewConsensusVariants.scala (hunk 2 above)
Comment (nh13) [reply src/main/scala/com/fulcrumgenomics/umi/ReviewConsensusVariants.scala]: The latter is the problem I had, which was `variant.filters` sometimes returns `Some("")`, so I have changed it to return "PASS"

**Comment by nh13** [comment]
back to you @tfenne

**Pull request merged**